src/listContext.js

```diff
@@ -109,23 +104,39 @@ const ListContextProvider = props => {
         numberOfPurchases,
       ),
     };
-
     return updateItem(item, purchase);
   };
 
+  //we use .set to add the time the item was purchased to an already existing document we search for the item.id
```

Comment:
I think this function an be removed. We dropped it in the last PR in favor of `purchaseItem`

Reply:
I tried to remove it but it was called in `purchaseItem`